.azure/pipelines/quarantined-tests.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       displayName: Build shared fx
     - script: .\restore.cmd -ci /p:BuildInteropProjects=true
       displayName: Restore interop projects
-    - script: .\build.cmd -ci -NoRestore -test -noBuildJava -all -projects eng\helix\helix.proj /p:RunQuarantinedTests=true /p:IsRequiredCheck=true /p:IsHelixJob=true /p:BuildInteropProjects=true /p:RunTemplateTests=true /p:ASPNETCORE_TEST_LOG_DIR=artifacts/log -bl
```

Comment:
This one is on purpose, we don't run java tests in quarantine

Reply:
Java tests can't be quarantined?

Reply:
Not right now. If we want to quarantine one we skip it for now. Currently there aren't any flaky ones so...